frontend/index.js

```diff
@@ -0,0 +1,155 @@
+let enrollStudentDiv = document.querySelector("#enrollStudentDiv")
+let addClassForm = document.querySelector("#addClassForm")
+let enrollStudentForm = document.querySelector("#enrollStudentForm")
+let sClassName = document.querySelector("#sClassName")
+let studentName = document.querySelector("#studentName")
+let studentAge = document.querySelector("#studentAge")
+let studentCity = document.querySelector("#studentCity")
+let studentGrade = document.querySelector("#studentGrade")
+let studenetEnrollBtn = document.querySelector("#studenetEnrollBtn")
+let newStudentDisplay = document.querySelector("#newStudentDisplay")
+let newStudentDisplayUL = document.querySelector("#newStudentDisplayUL")
+let failingForm = document.querySelector("#failingForm")
+let enterClass = document.querySelector("#enterClass")
+let addCity = document.querySelector("#addCity")
+let displayFailing = document.querySelector("#displayFailing")
+let fsBtn = document.querySelector("#fsBtn")
+let failingStudentList = document.querySelector("#failingStudentList")
+let failingStudentListUL = document.querySelector("#failingStudentListUL")
+// let res = url
+
+
+
+
+addClassForm.addEventListener("submit", async (event) => {
+    e.preventDefault()
+    // let className = document.querySelector("#className")
+    // let teacherName = document.querySelector("#teacherName")
+    ul.innerHTML = "";
+    let res = await axios.post(`http://localhost:3000/class`, {name: className, teacher: teacherName})
+    let h3 = document.createElement("h3");
+    h3.innerText = res.data.message
+    // `${req.body.name} was added to the list of classes being taught at Hogwarts. ${req.body.teacher} is leading the class.`
+    newClassDisplay.appendChild(h3)
+
+    // addClassForm.reset()  
+  })
+
+  enrollStudentForm.addEventListener("submit", async (e) => {
+    e.preventDefault()
+    let sClassName = document.querySelector("#sClassName").value
+
+    let name = document.querySelector("#studentName").value
+    let age = document.querySelector("#studentAge").value
+    let city = document.querySelector("#studentCity").value
+    let grade = document.querySelector("#studentGrade").value
+    let res = await axios.post("http://localhost:3000/class/:className/enroll",{class: sClassName.value, name: name.value, age: age.value, grade: grade.value, city: city.value})
+    let h3 = document.createElement("h3")
+    h3.innerText = res.data.message
+    // `${req.body.age} year old ${req.body.name} from ${req.body.city}is now enrolled in the ${className} class with a GPA of ${req.body.grade}.`
+    newStudentDisplay.appendChild(h3)
+  })
+
+    
+ 
+  
+    // let failingForm = document.querySelector("#failingForm");
+    // let addCity = document.querySelector("#addCity");
+    // let displayFailing = document.querySelector("#displayFailing");
+    failingForm.addEventListener("submit", e => {
+      e.preventDefault();
+      addClassFormm.innerHTML = "";
```

Comment:
You need to do this at the end of the function after you got all the information and used it to make a request. Otherwise your form inputs are being emptied immediately after submit.